README.rst

```diff
@@ -247,27 +247,20 @@ Formatting the number with currency:
 Testing
 -------
 
-Install the required packages:
+Get the source code:
 
 ::
 
     git clone https://github.com/django-money/django-money
 
     cd ./django-money/
 
-    pip install -e .[tests] # installation with required packages for testing
-
 Recommended way to run the tests:
 
 .. code:: bash
 
     tox
 
-Testing the application in the current environment python:
-
-.. code:: bash
-
-    make test
```

Comment:
Not sure, why remove the test target? It's very convenient to be able to run tests quickly without invoking tox...?